api/v1alpha1/backendtrafficpolicy_types.go

```diff
@@ -90,6 +107,18 @@ type ProtocolUpgradeConfig struct {
 	// TODO: support more options for CONNECT
 }
 
+type BufferRequests struct {
+	// MaxBufferSize specifies the maximum allowed size in bytes for each incoming request buffer.
+	// If exceeded, the request will be rejected with HTTP 413 Content Too Large.
+	//
+	// Accepts values in resource.Quantity format (e.g., "10Mi", "500Ki").
+	//
+	// +kubebuilder:validation:XIntOrString
+	// +kubebuilder:validation:Pattern="^[1-9]+[0-9]*([EPTGMK]i|[EPTGMk])?$"
+	// +notImplementedHide
+	MaxBufferSize resource.Quantity `json:"maxBufferSize,omitempty"`
```

Comment:
the word `buffer` is used twice `bufferRequests.maxBufferSize`
can we do better here ? maybe `bufferRequests.Limit` taking some inspiration from https://gateway.envoyproxy.io/docs/api/extension_types/#clientconnection

Reply:
Yer I agree with this for some consistency. The word `Limit` is also perhaps more k8s native as well